### DISCUSSION TITLE
Certificate Validation Issue

### DISCUSSION THREAD

**Discussion by kdavis5** [Budibase/budibase#4673] [Help]
Hi folks,
I am new to a lot of this so hopefully I will make sense. I am getting an error when running a REST query. "Unable to verify first certificate" I am using a private CA so I assume it needs to be trusted either in docker, on the host, or/and in budibase. I have read as much as possible but I am not able to get the query to run. Is there a way to disable validation? Thank you!

**Comment by mjashanks** [comment]
Just to check - can you successfully query your API from the host machine - i.e. the main OS where docker is running? I.e. by using a tool such as `curl` or Postman.

**Comment by kdavis5** [comment]
Hello,

Both worked but I had to disable cert verification on Postman. I put the internal CA cert chain on the host and it started working. Ran update-ca-certificates on the host and curl was then fine.

Still get the error on Budibase. Even went into the nginx docker container and ran an update-ca-certificates.

Strange, so I ended up tossing the same certificate I am using on the API machine on my F5 load-balancer and now Budibase is fine querying that. Just did that moments ago. Must be the server hosting the API, It is a Cisco product………

So what does Budibase use as the https client when sending these requests?

I really appreciate the response!

**Reply by mjashanks** [reply]
Great :)

Budibase uses the [node-fetch](https://www.npmjs.com/package/node-fetch) library
